scripts/zkai_claim_audit_comparison_artifacts_gate.py

```
@@ -0,0 +1,917 @@
+    if final and set(payload) != expected:
+        raise ClaimAuditError("payload field drift")
+    if payload.get("schema") != SCHEMA:
+        raise ClaimAuditError("schema drift")
+    if payload.get("decision") != DECISION:
+        raise ClaimAuditError("decision drift")
+    if payload.get("result") != RESULT:
+        raise ClaimAuditError("result drift")
+    validate_non_claim_inventory(payload)
+    source_artifacts = require_list(payload.get("source_artifacts"), "source artifacts")
+    if len(source_artifacts) != len(SOURCE_PATHS):
+        raise ClaimAuditError("source artifact count drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)